github-monitor/Ahk.GitHub.Monitor/Ahk.GitHub.Monitor.csproj

```diff
@@ -24,17 +24,17 @@
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="Polly" Version="7.2.2" />
+        <PackageReference Include="Polly" Version="7.2.3" />
         <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.354">
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
         </PackageReference>
         <PackageReference Include="Azure.Storage.Queues" Version="12.8.0" />
         <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.1.0" />
         <PackageReference Include="Microsoft.Extensions.Azure" Version="1.1.1" />
-        <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.15.0" />
-        <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="6.0.0" />
-        <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.0.1" />
+        <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.16.0" />
+        <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="6.0.1" />
+        <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.1.0" />
```

Comment:
Ne emelj verziót. Nem tudod, milyen hatása lehet. Még ha le is fordul a kód, akkor is, megváltozhat a viselkedés. A verzióemelés külön tesztelést igényel.

Reply:
done